src/app/static/src/tests/components/header/dataExplorationHeader.test.ts

```diff
@@ -58,13 +60,13 @@ describe(`Data Exploration header`, () => {
         const store = createStore({currentUser});
         const wrapper = getWrapper(currentUser, store);
         const loginInfo = wrapper.find("span");
-        expectTranslated(loginInfo, "Logged in as someone@email.com",
+        expectTranslatedWithStoreType(loginInfo, "Logged in as someone@email.com",
             "Connecté en tant que someone@email.com", "Sessão iniciada como someone@email.com", store);
     });
 
     it("renders language menu", () => {
         const wrapper = getWrapper()
-        expect(wrapper.findAll(LanguageMenu).length).toBe(1);
+        expect(wrapper.find("language-menu-stub").exists()).toBe(true)
```

Comment:
`exists` is neater than length check on findAll, but could you not use `find(LanguageMenu)` rather than `find("language-menu-stub")`?